pkgs/top-level/python-packages.nix

```diff
@@ -11387,34 +11392,43 @@ in {
   };
 
   nipype = buildPythonPackage rec {
-    version = "0.10.0";
+    version = "0.14.0";
     name = "nipype-${version}";
 
-    # Uses python 2 print. Master seems to be Py3 compatible.
-    disabled = isPy3k;
-
     src = pkgs.fetchurl {
       url = "mirror://pypi/n/nipype/${name}.tar.gz";
-      sha256 = "7fb143cd4d05f18db1cb7f0b83dba13d3dcf55b4eb3d16df08c97033ccae507b";
+      sha256 = "0airdrh93vwmbfkqxp5cqfzm0zzqcvjnvphv3zhg197y39xxpl1k";
     };
 
-    # Tests fail due to getcwd returning ENOENT???
+    # Fails in testing, "no such file or directory: which"
```

Comment:
Have you tried adding the `which` package?

Reply:
I didn't realise which was a command :s
`which which` gives `which: built in shell command`

Adding which actually does help, but still fails :(

Reply:
in `zsh` it is a builtin.